### PR TITLE
Some changes to the changelog

### PR DESCRIPTION
**Pull request by LyesSaadi**

### CODE REVIEW

File: data/com.github.GradienceTeam.Gradience.appdata.xml.in.in [hunk 1]
```diff
@@ -38,11 +38,11 @@
...
       <description>
-        <p>Gradience 0.3.0 is a major update with many new features and improvements. here is some of them:</p>
+        <p>Gradience 0.3.0 is a major update with many new features and improvements. Here are some of them:</p>
         <ul>
-          <li>Added plugins support, this allows creating plugins for customizing other apps</li>
-          <li>Added support for custom repos, this allows creating own selection of presets</li>
-          <li>Preset Manager performance are significantly enhanced, presets are downloading much faster and app don't freeze on preset removal</li>
+          <li>Added plugins support, this allows the creation of plugins to customize other apps</li>
+          <li>Added support for custom preset repos, this allows users to host a collection of presets on their own infrastructure</li>
+          <li>Preset Manager performance are significantly enhanced, presets are downloaded faster and the app doesn't freeze on preset removal</li>
```
Comment (daudix-UFO) [inline data/com.github.GradienceTeam.Gradience.appdata.xml.in.in:45]: downloads as this is not in the past

**Comment by daudix-UFO** [comment]
Last review

**Comment by daudix-UFO** [comment]
Nice 👍

**Comment by 0xMRTT** [comment]
Funny PR name 😂

**Pull request merged**